jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -647,4 +656,29 @@ private void writeTarFile(
   private <E> List<Future<E>> scheduleCallables(ImmutableList<? extends Callable<E>> callables) {
     return callables.stream().map(executorService::submit).collect(Collectors.toList());
   }
+
+  private String computeArchitecture(String architecture) {
+    if (architecture.equals("x86_64")) {
```

Comment:
Where do we get the mapping between the architecture from docker info and the architecture needed for docker build? Are `x86_64 ` and `aarch64 ` all we need? Also please add unit tests for all the cases.

Reply:
I see, so the root cause is that the architecture returned from `docker info` is different from the architecture used by `docker build`. If that's the case, maybe we can change the method name to something like `mapDockerInfoArchitechtureToDockerBuildArchitechture`? Or `toDockerBuildArchitecture(String dockerInfoArchitecture)` if we think the previous one is too verbose?

Reply:
That's right, the naming doesn't seem to be standardized everywhere. While `docker info` results in `x86_64` on my machine, `docker manifest inspect` on images uses `amd64` which is supposed to be equivalent (https://github.com/docker/docs/issues/4295). 
Renaming the method is a good idea. Alternatively, what are your thoughts on `normalizeArchitecture()` or `standardizeArchitecture()`? I found a couple of references that do something similar: 
(1)https://github.com/containerd/platforms/blob/c1438e911ac7596426105350652fe267d0fb8a03/database.go#L76
 (2)https://github.com/openzipkin/zipkin/blob/2f0a26fd4e3f3ea24e9fb30f55a84ebbc194129c/build-bin/docker/docker_arch#L7

Reply:
Thanks for the suggestions! I like `normalizeArchitecture()`.